deploy/helm/scf/assets/operations/instance_groups/router.yaml

```diff
@@ -2,14 +2,17 @@
 - type: remove
   path: /instance_groups/name=router/jobs/name=loggr-udp-forwarder
 
+# Override the addresses for the jobs under the router instance group.
 - type: replace
   path: /instance_groups/name=router/jobs/name=gorouter/properties/router/dns_health_check_host?
   value: ((deployment-name))-uaa
-
 - type: replace
   path: /instance_groups/name=router/jobs/name=gorouter/properties/uaa/token_endpoint?
   value: ((deployment-name))-uaa
-
 - type: replace
   path: /instance_groups/name=router/jobs/name=gorouter/properties/routing_api/uri?
   value: https://((deployment-name))-api-group
```

Comment:
FWIW, this should be `api` not `api-group`; we just called it `api-group` in SCF v1 because we couldn't handle the conflict with the job name at the time.

Reply:
Thanks. This is from the first router commit. I didn't notice this.